### DISCUSSION TITLE
How to improve the perfomence when load a complicate web?

### DISCUSSION THREAD

**Discussion by marderary** [cefsharp/CefSharp#3543] [Q&A] [answered]
Compare with the native Chrome browser, it  seems to CefSharp wpf load web slowly even I can aware of the low speed by my eyes, any one knows how to fast these?

**Comment by amaitland** [comment, marked as answer]
The `WPF` version is slower than the `WinForms` or [CefSharp.Wpf.HwndHost](https://github.com/cefsharp/CefSharp.Wpf.HwndHost) versions as it is an actual `WPF` control rather than a `WindowsFormsHost`/`HwndHost`. See https://github.com/cefsharp/CefSharp/wiki/General-Usage#offscreen-rendering-osr

If raw performance is all you care about then switch to the `CefSharp.Wpf.HwndHost` version (doesn't support transparency etc).

By default `Site Isolation Trials` are currently disabled (issue https://github.com/cefsharp/CefSharp/issues/2967), so websites that have lots of frames of different origins will load slower.

You can remove this command line arg, look something like:

```c#
var cefSettings = new CefSettings();
settings.CefCommandLineArgs.Remove("disable-site-isolation-trials");
```

**Reply by marderary** [reply]
Thanks a lot ,I will have a try and shot a response when I get test result.
